scripts/test_reports/report.py

```diff
@@ -251,12 +269,20 @@ def parse_file(self, result_file):
                                 infra_issue = True
                             elif tc.result.type == 'error':
                                 test_status = self.status.BLOCKED
+                                if self.configbuginfo:
+                                  if self.configbuginfo.has_section(config):
+                                    if self.configbuginfo.has_option(config,ref):
+                                      cr['defects'] = self.configbuginfo.get(config, ref)
```

Comment:
wouldn't be cleaner something like:
```
if self.configbuginfo and self.configbuginfo.has_section(config) and
   self.configbuginfo.has_option(configm ref): 
```

Reply:
https://github.com/ceolin

Reply:
Hi Ceolin -- Thanks for your comments, I have updated the code again. So could you please review this PR again ? Thanks again.